recipes/libcamera/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "libcamera" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://git.libcamera.org/libcamera/libcamera.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  
+requirements:
+  build:
+    - cmake
+    - ninja
+    - meson
+    - pkg-config
+    - python
+    - pybind11
```

Comment:
I think python and pybind11 should oddly be moved to host.

Reply:
hmm I'm skeptical about this,  but let's try. I think it was ok in build.